text/0000-optimize-by-default.md

```diff
@@ -0,0 +1,166 @@
+- Start Date: 2015-03-10
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Make rustc and cargo produce optimized binaries by default.
+
+# Motivation
+
+
+Optimizing by default is a safety & usability feature that everyone benefits
```

Comment:
Just because "memory safety" is a common bigram in Rust parlance doesn't mean the word "safety" has been redefined. :P :)

By "unsafe", I mean "debug-by-default is a design prone to error and damage". Optimized-by-default is not, and is thus safer.